pb_test_configs.yml

```diff
@@ -6,41 +6,44 @@
   connection: local
 
   vars:
-    output_dir: "outputs"
+    output_dir: "tests/outputs"
     test_dir: "tests"
+    template_dir: "{% set env_var = lookup('env', 'TEMPLATE_DIR') %}{% if env_var %}{{ env_var }}{% else %}templates{% endif %}"

```

Comment:
I know you've already merged this, but can you please circle back to update the README to document use of this environment variable?